manifests/Zeal/Zeal/0.6.1.yaml

```diff
@@ -0,0 +1,15 @@
+Id: Zeal.Zeal
+Name: Zeal
+Publisher: zealdocs.org
+Version: 0.6.1
+AppMoniker: zeal
+Description: Zeal is an offline documentation browser for software developers
+Homepage: https://zealdocs.org/
+License: GPLv3
+LicenseUrl: https://www.gnu.org/licenses/gpl-3.0.html
+InstallerType: msi
+Installers:
+  - Arch: x64
+    Url: https://dl.bintray.com/zealdocs/windows/zeal-0.6.1-windows-x64.msi
+    Sha256: 875a91b0aa91a7dc56ef23b551006fd64b33ebfb16b710e4924c391751e6d515
+ManifestVersion: 0.1.0
```

Comment:
```suggestion

```
[Despite the current spec saying that this field is required](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv0.1.md), it's currently not even supported.